### PR TITLE
[MISC] Update the simd benchmark to use correct alignment output conf…

### PR DESCRIPTION
**Pull request by rrahn**
Follow up of changing to the output configuration of the alignment.

### CODE REVIEW

**Comment by rrahn** [comment]
ci error not related:
```
99% tests passed, 1 tests failed out of 283

Total Test time (real) =  23.68 sec

The following tests FAILED:
	129 - argument_parser/detail/version_check_release_test (SEGFAULT)
Errors while running CTest
##[error]Process completed with exit code 8.
```

File: test/performance/alignment/global_affine_alignment_simd_benchmark.cpp [hunk 1]
```diff
@@ -78,28 +78,34 @@ void seqan3_affine_dna4_accelerated(benchmark::State & state, align_configs_t &&
...
                   seqan3::align_cfg::result{seqan3::with_score, seqan3::using_score_type<int16_t>},
+                  seqan3::align_cfg::output_score,
                   seqan3::align_cfg::vectorised)
                         ->UseRealTime()
                         ->DenseRange(deviation_begin, deviation_end, deviation_step);
 
 BENCHMARK_CAPTURE(seqan3_affine_dna4_accelerated,
                   simd_with_end_position,
-                  seqan3::align_cfg::result{seqan3::with_end_positions, seqan3::using_score_type<int16_t>},
+                  seqan3::align_cfg::result{seqan3::with_score, seqan3::using_score_type<int16_t>},
```
Comment (marehr) [inline test/performance/alignment/global_affine_alignment_simd_benchmark.cpp:88]:
```suggestion
                  seqan3::align_cfg::result{seqan3::with_end_positions, seqan3::using_score_type<int16_t>},
```

Comment (rrahn) [inline test/performance/alignment/global_affine_alignment_simd_benchmark.cpp:88]:
```suggestion
                  seqan3::align_cfg::result{seqan3::with_end_positions, seqan3::using_score_type<int16_t>},
```

File: test/performance/alignment/global_affine_alignment_simd_benchmark.cpp [hunk 2]
```diff
@@ -78,28 +78,34 @@ void seqan3_affine_dna4_accelerated(benchmark::State & state, align_configs_t &&
...
+                  seqan3::align_cfg::output_score,
                   seqan3::align_cfg::vectorised,
                   seqan3::align_cfg::parallel{get_number_of_threads()})
                         ->UseRealTime()
                         ->DenseRange(deviation_begin, deviation_end, deviation_step);
 
 BENCHMARK_CAPTURE(seqan3_affine_dna4_accelerated,
                   simd_parallel_with_end_position,
-                  seqan3::align_cfg::result{seqan3::with_end_positions, seqan3::using_score_type<int16_t>},
+                  seqan3::align_cfg::result{seqan3::with_score, seqan3::using_score_type<int16_t>},
```
Comment (marehr) [inline test/performance/alignment/global_affine_alignment_simd_benchmark.cpp:106]:
```suggestion
                  seqan3::align_cfg::result{seqan3::with_end_positions, seqan3::using_score_type<int16_t>},
```

**Review by marehr — CHANGES_REQUESTED** [review]

**Review by marehr — APPROVED** [review]

**Review by smehringer — APPROVED** [review]
LGTM

**Comment by smehringer** [comment]
I can't merge from my phone :D feel free to merge.

**Pull request merged**